media/video-stitcher/test/stitcher.test.js

```diff
@@ -116,6 +121,82 @@ before(() => {
       }
     });
 
+  // Delete existing test VOD configs more than an hour old.
+  const vodConfigs = execSync(
+    `node listVodConfigs.js ${projectId} ${location}`,
+    {cwd}
+  );
+
+  vodConfigs
+    .toString()
+    .split(/\r?\n/)
+    .forEach(line => {
+      if (
+        line.includes(`locations/${location}/vodConfigs/${vodConfigIdPrefix}`)
+      ) {
+        this.nextId = line.split('/').pop();
```

Comment:
For clarity, rather than attach to the `this` context, declare `nextId` instead (please apply the change to the other tests as well).

Reply:
Done for `nextId`.